tests/integration/docusaurus/tutorials/quickstart/quickstart_v1__1.py

```diff
@@ -0,0 +1,79 @@
+"""
+This guide demonstrates how to use Great Expectations in a quick-start
+notebook environment. It is useful to explore and understand how Great Expectations
+works, using your own data.
+
+Note:
+- Do not follow this workflow when you are building a durable workflow because it uses
+ephemeral assets.
+- Do not use this workflow for embedding in a pipeline or CI system, because it uses an
+iterative process for trying and refining expectations.
+"""
+
+from enum import Enum
+
+# <snippet name="tutorials/quickstart/quickstart.py import_gx">
+import great_expectations as gx
+import great_expectations.expectations as gxe
+
+# </snippet>
+
+# Set up
+# NOTE: Context is a singleton now. Once the context has been set instantiated in a session
+# <snippet name="tutorials/quickstart/quickstart.py get_context">
+context = gx.get_context()
+# </snippet>
+
+
+class QuickstartDatasourceTabs(Enum):
+    PANDAS_DEFAULT = "pandas_default"
+    SQL_QUERY = "sql_query"
+
+
+# TODO: Where in the GX namespace does Batch live?
+def get_quickstart_batch(datasource_type: QuickstartDatasourceTabs) -> Batch:
+    if datasource_type == QuickstartDatasourceTabs.PANDAS_DEFAULT:
+        # <snippet name="tutorials/quickstart/quickstart.py connect_to_data pandas_csv">
+        batch = context.sources.pandas_default.read_csv(
+            "https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv"
+        )
+        # </snippet>
+        return batch
+
+    elif datasource_type == QuickstartDatasourceTabs.SQL_QUERY:
+        # TODO: add local postgresql db to quickstart? or add public snowflake?
+        connection_string = "postgresql://postgres:postgres@localhost:5432/postgres"
+        # <snippet name="tutorials/quickstart/quickstart.py connect_to_data sql_query">
+        datasource = context.sources.add_postgresql_datasource(
+            name="quickstart_db",
+            connection_string=connection_string,
+        )
+        batch = datasource.read_query("SELECT * FROM yellow_tripdata_sample_2019_01")
+        # </snippet>
+        return batch
+
+
+for tab_name in QuickstartDatasourceTabs:
+    batch = get_quickstart_batch(datasource_type=tab_name)
+
+# <snippet name="tutorials/quickstart/quickstart.py create_expectation">
+expectation = gxe.ExpectColumnValuesToNotBeNull(
```

Comment:
We will need to use keyword arguments for Expectation instantiation.

Reply:
@cdkini File a ticket for non-keyword arg domain args.

Reply:
roger